Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -51,18 +51,27 @@ extension NavigationMapView {
      
      - parameter routeProgress: The current `RouteProgress`.
      - parameter coordinate: The current user location coordinate.
+     - parameter redraw: A `Bool` value to decide whether the route is new. When style changes, `RouteController` did refresh route or reroute, the value
+     should be set to `true`. When `RouteController` did update the `RouteProgress`, the value should be set to `false`.
      */
-    func updateRouteLine(routeProgress: RouteProgress, coordinate: CLLocationCoordinate2D?) {
-        show([routeProgress.route], legIndex: routeProgress.legIndex)
-        guard routeLineTracksTraversal else { return }
+    public func updateRouteLine(routeProgress: RouteProgress, coordinate: CLLocationCoordinate2D?, redraw: Bool = false) {
```

Comment:
What about?
```suggestion
    public func updateRouteLine(routeProgress: RouteProgress, coordinate: CLLocationCoordinate2D?, shouldRedraw: Bool = false) {
```

Reply:
Thanks for the suggestion, the method name has been updated.